static_src/util/format_date.js

```diff
@@ -1,18 +1,17 @@
 import moment from 'moment-timezone';
 
-const invalidMsg = 'Invalid datetimes cannot be formatted.';
+const makeInvalidDateTimeValueError = str =>
+  new Error(`could not format invalid date/time value: ${str}`);
 
-/**
- * Returns a formated datetime string
- * @params {String} datetime as a string
- * @params {String} timezone abbreviation - defaults to UTC
- *
- * @returns {String} A "MM/DD/YYYY H:mma z" formatted string
- *                    i.e. 03/21/2016 10:39am PDT
-*/
-export default function formatDateTime(dateString, timezone = 'Etc/UTC') {
-  const d = moment(dateString);
+const formatDateTime = (str, timezone = 'Etc/UTC') => {
```

Comment:
`str` -> `dateString`

Reply:
I named it `str` because technically the caller can pass any string that is not a date, e.g. `formatDateTime('random non date string')` and `str` encapsulates that an arbitrary string was passed. Per https://golang.org/src/time/format.go?s=23626:23672#L762

I can change it though.

Reply:
Thanks for that. I can see the case for leaving it as str. Ignore this and just bring back the comment